src/components/AddRpcButton.tsx

```diff
@@ -40,8 +43,22 @@ const ERROR_NETWORK_ADDING_UNSUPPORTED = { errorMessage: `Oh no! 😢 It looks l
 const ERROR_NETWORK_ALREADY_ADDED = { errorMessage: `Your wallet has a pending request to add the network. Please review your wallet.`, isUserRejection: false, isError: true }
 
 function getErrorMessage(error: any): { errorMessage: string | null, isUserRejection: boolean, isError: boolean } {
```

Comment:
Since the function is big enough, I suggest to move it into a separate file

Reply:
https://github.com/mevblocker/web/pull/64